From 55f26470f6efd0eb1fcbfd4caa2c11c370d8ee73 Mon Sep 17 00:00:00 2001
From: "dongwan0605.kim" <dongwan0605.kim@lge.com>
Date: Fri, 3 Feb 2017 10:01:18 +0900
Subject: [PATCH 14/31] Better QS detail clip animation

Clip animation could be started with panel expanding.
Animation's target height should be expanded panel height.

Test: Click QQS which has detail view (ex. Battery tile)
 > Check animation while panel is expanding

Change-Id: Ia7b5a5b80739c50c0b09bd519de60f7e8c06a55c
---
 .../SystemUI/src/com/android/systemui/qs/QSDetailClipper.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java b/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java
index c454048d064..2ca1e621009 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/QSDetailClipper.java
@@ -40,8 +40,8 @@ public class QSDetailClipper {
         if (mAnimator != null) {
             mAnimator.cancel();
         }
-        final int w = mDetail.getWidth() - x;
-        final int h = mDetail.getHeight() - y;
+        final int w = mDetail.getMeasuredWidth() - x;
+        final int h = mDetail.getMeasuredHeight() - y;
         int innerR = 0;
         if (x < 0 || w < 0 || y < 0 || h < 0) {
             innerR = Math.abs(x);
-- 
2.17.1

